src/components/Datasets.js

```diff
@@ -0,0 +1,255 @@
+    }
+  };
+
+  return (
+    <Main content>
+      <Container>
+        <h1>Datasets</h1>
+        <Table aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              <TableCell align="left">
+                <TableSortLabel
+                  active={orderBy === DATASETS_TABLE_COLUMNS.NAME}
+                  direction={
+                    orderBy === DATASETS_TABLE_COLUMNS.NAME && !isAsc
+                      ? ORDER_BY.DESC
+                      : ORDER_BY.ASC
+                  }
+                  onClick={() => {
+                    handleSortColumn(DATASETS_TABLE_COLUMNS.NAME);
+                  }}
+                >
+                  <strong>{t('Name')}</strong>
+                </TableSortLabel>
+              </TableCell>
+              <TableCell align="right">
+                <TableSortLabel
+                  active={orderBy === DATASETS_TABLE_COLUMNS.SIZE}
+                  direction={
+                    orderBy === DATASETS_TABLE_COLUMNS.SIZE && !isAsc
+                      ? ORDER_BY.DESC
+                      : ORDER_BY.ASC
+                  }
+                  onClick={() => {
+                    handleSortColumn(DATASETS_TABLE_COLUMNS.SIZE);
+                  }}
+                >
+                  <strong>{t('Size')}</strong>
+                </TableSortLabel>
+              </TableCell>
+              <TableCell align="right">
+                <TableSortLabel
+                  active={orderBy === DATASETS_TABLE_COLUMNS.CREATED}
+                  direction={
+                    orderBy === DATASETS_TABLE_COLUMNS.CREATED && !isAsc
+                      ? ORDER_BY.DESC
+                      : ORDER_BY.ASC
+                  }
+                  onClick={() => {
+                    handleSortColumn(DATASETS_TABLE_COLUMNS.CREATED);
+                  }}
+                >
+                  <strong>{t('Created')}</strong>
+                </TableSortLabel>
+              </TableCell>
+              <TableCell align="right">
+                <TableSortLabel
+                  active={orderBy === DATASETS_TABLE_COLUMNS.LAST_MODIFIED}
+                  direction={
+                    orderBy === DATASETS_TABLE_COLUMNS.LAST_MODIFIED && !isAsc
+                      ? ORDER_BY.DESC
+                      : ORDER_BY.ASC
+                  }
+                  onClick={() => {
+                    handleSortColumn(DATASETS_TABLE_COLUMNS.LAST_MODIFIED);
+                  }}
+                >
+                  <strong>{t('Last Modified')}</strong>
+                </TableSortLabel>
+              </TableCell>
+              <TableCell align="right" />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {stableSort(fakeDatasets, getComparator(isAsc, orderBy)).map(
+              (dataset) => (
+                <TableRow key={dataset.name}>
+                  <TableCell>
+                    <span style={{ fontSize: '20px' }}>{dataset.name}</span>
```

Comment:
use `<Typography>` instead of `<span>`, and avoid `</br>` since it might render differently on different browsers. Also, strong preference is to declare any styles in the `makeStyles` hook rather than inline